pkgs/by-name/im/imsprog/package.nix

```
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitHub, cmake, makeWrapper, pkg-config, bash, gnome, libusb1, qt5, wget }:
+
+stdenv.mkDerivation rec {
+  pname = "IMSProg";
+  version = "1.3.9";
+
+  src = fetchFromGitHub {
+    owner = "bigbigmdm";
+    repo = "IMSProg";
+    rev = "v" + version;
```

```suggestion
    rev = "refs/tags/v${version}";
```
Including the full ref is often preferred.